templates/dashboard.yaml

```diff
@@ -1,3 +1,6 @@
+# Optionally include the dashboard if it's specified.
+# The dashboard will be removed in v1.21.
```

Comment:
We may need to keep the old kube Service around and just point it to coderd or something to help keep shit from breaking for people who don't heed the warnings. Idk if we should in this PR or just figure that out separately towards release

Reply:
Agreed agreed.

If they exclude `dashboard` from the `values.yaml`, it just won't deploy it which is great!